app/src/main/java/com/example/cactusnotes/validations/UsernameValidator.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.cactusnotes.validations
+
+import com.example.cactusnotes.R
+import com.example.cactusnotes.ValidationResult
+
+class UsernameValidator : Validator {
+    override fun validate(field: String)= when {
```

Comment:
This file is not properly formatted.

You can check *Reformat code* on commit settings to enable auto-format before each commit.